src/core/api/index.js

```diff
@@ -20,7 +20,15 @@ export const category = new Entity('categories', {}, { idAttribute: 'slug' });
 export const user = new Entity('users', {}, { idAttribute: 'username' });
 
 export function makeQueryString(query) {
-  return url.format({ query });
+  const resolvedQuery = { ...query };
+  Object.keys(resolvedQuery).forEach((key) => {
+    if (resolvedQuery[key] === undefined) {
```

Comment:
Whoops, good call. I also synced up with mat on what DRF expects for boolean values.